Automattic/jetpack
# Super Cache: Overhaul styling to match Jetpack aesthetics

Retry of #26890. cc @mikestottuk 

## Proposed changes:

![After Easy 2](https://github.com/Automattic/jetpack/assets/4044428/7f564f5b-93d4-418b-bb09-b4d920fb512d)

* Adds a new `dashboard.css` which introduces styling for:
    * A new header, footer, navigation component, and card component.
* Applies these new styles to all known Super Cache pages:
    * Easy, Advanced, CDN, Contents, Preload, Plugins, and Debug pages.

## Screenshots

Page|Before|After
-|-|-
Easy|![Before Easy](https://github.com/Automattic/jetpack/assets/4044428/adab503d-c4fd-4f0b-a3d7-5d4511dd7185)|![After Easy](https://github.com/Automattic/jetpack/assets/4044428/4e3adfb4-ffc7-4187-8f38-bd49af5aa6ea)
Advanced|![Before Advanced](https://github.com/Automattic/jetpack/assets/4044428/49c7af8b-e8ee-46f5-b6e0-7b4adec82185)|![After Advanced](https://github.com/Automattic/jetpack/assets/4044428/30d278d3-4bcd-4d68-9420-d7130172ab93)
CDN|![Before CDN](https://github.com/Automattic/jetpack/assets/4044428/9fd56d6c-a291-4c54-9984-370b2768f132)|![After CDN](https://github.com/Automattic/jetpack/assets/4044428/e9db232b-3ec7-43b3-8b6b-ca7d8753a3ac)
Contents|![Before Contents](https://github.com/Automattic/jetpack/assets/4044428/eb21b549-0861-4307-807f-a2e92ae12ab6)|![After Contents](https://github.com/Automattic/jetpack/assets/4044428/be10cdac-cfd2-4924-8a4f-58fdc7f7cbf8)
Preload|![Before Preload](https://github.com/Automattic/jetpack/assets/4044428/31ef0fda-3bb2-44e8-9c71-5b7473eeb3fd)|![After Preload](https://github.com/Automattic/jetpack/assets/4044428/e86ed58c-5591-4dd7-9e5f-48f30fe629f8)
Plugins|![Before Plugins](https://github.com/Automattic/jetpack/assets/4044428/fcf1a433-3e1e-4585-9835-ac1c3987e25e)|![After Plugins](https://github.com/Automattic/jetpack/assets/4044428/904b411b-8601-4dae-9ed0-d3eae7736cf2)
Debug|![Before Debug](https://github.com/Automattic/jetpack/assets/4044428/3def11bf-0d31-4f1f-b102-cdf873ed1137)|![After Debug](https://github.com/Automattic/jetpack/assets/4044428/4006f5c7-8f6d-488d-b10f-20466c6a473f)

## Other information:


## Jetpack product discussion


Prior discussion: pc9hqz-1e4-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Apply these changes to your test site.
* Ensure that the new styling has been applied to each Super Cache settings tab.
